omnibus/config/software/datadog-puppy.rb

```diff
@@ -50,6 +51,41 @@
           vars: { install_dir: install_dir }
     end
   end
+  if windows?
```

Comment:
If I understand correctly, with the current state of this file, the Linux puppy Agent contains the main Agent, but the Windows puppy Agent contains the main Agent + trace + process. Is this difference intended?

Reply:
I was unaware that the linux puppy didn't include the other services.  I thought that the main idea of the puppy was `no python`.